### ISSUE TITLE
Step indicator has an `<aria-label>` on element that does not support it

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4026]
## Area or Component
Sign up form

## Quick Summary
The progress bar at the top is a `<div>` element with an `aria-label`. `aria-label` is not supported for `div`.

## Accessibility Violation
Elements must only use allowed ARIA attributes: https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome

## Action Requested
Either change the element type or remove `aria-label`. 
Related is ticket #4013; it may help to look at both tickets at the same time.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/a520f60ba21ab1458b3093fbdfb96e61f40101f4/frontend/src/app/commonComponents/StepIndicator.tsx#L79-L79

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=37:37
